src/plugins/interactive-media/interactive-media.yml

```diff
@@ -46,6 +46,18 @@ continue:
 replay:
   en: Replay
   de: Wiederholen
+end:
+  en: End
+  de: Enden
 reset:
   en: Reset
   de: Zurücksetzen
+results:
+  en: Results
+  de: Ergebnisse
+currentProgress:
+  en: Your current progress
+  de: Dein aktueller Fortschritt
+resultsSummary:
+  en: "{answerCount} out of {totalCount} answered, out of which {correctCount} correctly and {incorrectCount} incorrectly."
+  de: "{answerCount} von {totalCount} Fragen beantwortet, aus denen {correctCount} richtig und {incorrectCount} falsch."
```

Comment:
"davon" instead of "aus denen"

Reply:
Ich hatte 7 Wörte auf Deutsch geschrieben, davon 2 falsch 😂 Korrigiert, danke!